### PR TITLE
Support for Dark Theme in Checkbox

### PR DESCRIPTION
**Pull request by pooja-ps-trimble**

### CODE REVIEW

File: Trimble.Modus.Components/Trimble.Modus.Components.csproj [hunk 1]
```diff
@@ -1,273 +1,268 @@
...
-	  <Compile Update="Controls\DropDown\DropDownContents.xaml.cs">
-	    <DependentUpon>DropDownContents.xaml</DependentUpon>
-	  </Compile>
-	  <Compile Update="Controls\TMBadge\TMBadgeStyles.xaml.cs">
-	    <DependentUpon>TMBadgeStyles.xaml</DependentUpon>
-	  </Compile>
-	  <Compile Update="Controls\TMCard\TMCard.xaml.cs">
-	    <DependentUpon>asas.xaml</DependentUpon>
-	  </Compile>
-	  <Compile Update="Controls\TMChips\TMChipStyles.xaml.cs">
```
Comment (rthanga1) [inline Trimble.Modus.Components/Trimble.Modus.Components.csproj:160]: all these intentional?

File: Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs [hunk 2]
```diff
@@ -140,14 +142,14 @@ private static void OnIsCheckedChanged(BindableObject bindable, object oldValue,
             var customCheckboxView = (TMCheckBox)bindable;
             if (customCheckboxView.IsChecked)
             {
-                customCheckboxView._checkbox.Source = ImageSource.FromFile(ImageConstants.CheckedCheckBoxButton);
-
+                customCheckboxView._checkbox.SetAppTheme<FileImageSource>(
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs:145]: Here also, try the VisualStateManager. The checkbox can have Normal/Checked/Interminate states

**Review by rthanga1 — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs (hunk 2 above)
Comment (pooja-ps-trimble) [reply Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs:145]: As TMChips doesn't have a xaml I wasn't able to proceed with that approach. Is it possible without xaml?

Comment (rthanga1) [reply Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs:145]: Yes. You can still have a TMChipsStyles.xaml and use it in code

File: Trimble.Modus.Components/Trimble.Modus.Components.csproj [hunk 3]
```diff
@@ -122,13 +121,10 @@
     <MauiXaml Update="Controls\Button\TMButton.xaml">
       <Generator>MSBuild:Compile</Generator>
     </MauiXaml>
-    <MauiXaml Update="Controls\ProgressBar\TMProgressBarStyles.xaml">
```
Comment (rthanga1) [inline Trimble.Modus.Components/Trimble.Modus.Components.csproj:125]: Required?

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**